error in npm install DaftMonk/grunt-build-control

I have an error generating a project, in npm install, i have the next error:
```
Error: No compatible version found: grunt-build-control@'DaftMonk/grunt-build-control'
npm ERR! Valid install targets:
npm ERR! ["0.0.1","0.1.0","0.1.1","0.1.2","0.1.3","0.1.4","0.1.5","0.1.6","0.1.7","0.1.8","0.2.0","0.2.1","0.2.2","0.3.0"]
npm ERR!     at installTargetsError (/usr/share/npm/lib/cache.js:719:10)
npm ERR!     at /usr/share/npm/lib/cache.js:638:10
npm ERR!     at saved (/usr/share/npm/node_modules/npm-registry-client/lib/get.js:142:7)
npm ERR!     at /usr/lib/nodejs/graceful-fs/polyfills.js:133:7
npm ERR!     at Object.oncomplete (fs.js:107:15)
```
I solved it installing grunt-build-control without an especific version, but i don't know if it will be conflicts with other packages.

Congrats for this excellent project!

Regards,
Luis

--- a/app/templates/_package.json
+++ b/app/templates/_package.json
@@ -60,7 +60,7 @@
     "grunt-protractor-runner": "^1.1.0",
     "grunt-injector": "~0.5.4",
     "grunt-karma": "~0.8.2",
-    "grunt-build-control": "DaftMonk/grunt-build-control",
+    "grunt-build-control": "~0.4.0",
     "grunt-mocha-test": "~0.10.2",<% if(filters.sass) { %>
     "grunt-contrib-sass": "^0.7.3",<% } %><% if(filters.stylus) { %>
     "grunt-contrib-stylus": "latest",<% } %>

